ui/components/app/detected-token/detected-token-ignored-popover/__snapshots__/detected-token-ignored-popover.test.tsx.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`DetectedTokenIgnoredPopover should match snapshot for ignore mode 1`] = `<DocumentFragment />`;
+
+exports[`DetectedTokenIgnoredPopover should match snapshot for import mode 1`] = `<DocumentFragment />`;
```

Comment:
These snapshots are empty, so they aren't testing anything.   I think modals have to be snapshotted differently, I'll see if i can find an example

Reply:
I think this does it:
https://github.com/MetaMask/metamask-extension/pull/29061/commits/0f5eda3569c726401371c338c410e9a358376a7a